resources/dicts/thoughts/alive/general.json

```
@@ -51,87 +49,1424 @@
+            "Sits quietly watching cats go about their day",
+            "Is wondering if there are clans outside of the ones around {PRONOUN/m_c/poss} area",
+            "Is wondering if Starclan is watching cats outside the clans",
+            "Let out a big yawn!",
+            "Is wondering if clouds are as soft as they look",
+            "Is enjoying the sounds of nature",
+            "Is feeling anxious",
+            "Is feeling restless",
+            "Is dozing off",
+            "Is a bit annoyed today",
+            "Today is just not m_c's day...",
+            "Is regretting not going to sleep sooner...",
+            "Is tidying up camp",
+            "Is tidying up camp",
+            "Just can't find anything appetizing on the fresh-kill pile",
+            "Has more energy then {PRONOUN/m_c/subject} know what to do with at the moment",
+            "Woke up on the wrong side of the nest",
+            "Is complaining about an itch {PRONOUN/m_c/subject} can't quite reach"
+        ],
+        "main_status_constraint": [
+            "kit",
+            "apprentice",
+            "medicine cat apprentice",
+            "mediator apprentice",
+            "warrior",
+            "medicine cat",
+            "mediator",
+            "deputy",
+            "leader",
+            "elder"
+        ]
+    },
+    {
+        "id": "gen_to_other",
+        "thoughts": [
+            "Is asking r_c an important question",
+            "Can't find r_c anywhere",
+            "Is fetching r_c for another cat who was looking for {PRONOUN/m_c/object}",
+            "Is worried r_c is overworking {PRONOUN/m_c/self}",
+            "Is finding r_c annoying",
+            "Can't keep up with r_c",
+            "Is playing mossball with r_c",
+            "Is eavesdropping on r_c",
+            "Wants to get to know r_c better",
+            "Is sharing tongues with r_c",
+            "Wonders how r_c is doing",
+            "Is giving some advice to r_c",
+            "Wonders what r_c is up to",
+            "Is playing a prank on r_c"
+        ],
+        "main_status_constraint": [
+            "kit",
+            "apprentice",
+            "medicine cat apprentice",
+            "mediator apprentice",
+            "warrior",
+            "medicine cat",
+            "mediator",
+            "deputy",
+            "leader",
+            "elder"
+        ],
+        "random_status_constraint":[
+            "any"
+        ],
+        "random_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "gen_not_kit",
+        "thoughts": [
+            "Is thinking on {PRONOUN/m_c/poss} life choices",
+            "Wishes {PRONOUN/m_c/subject} had the energy {PRONOUN/m_c/subject} had as a kit",
+            "Found a wonderful napping spot outside camp",
+            "Had a nightmare involving a Twoleg trap",
+            "Is humming a tune {PRONOUN/m_c/subject} heard near a twolegplace",
```

```suggestion
            "Is humming a tune {PRONOUN/m_c/subject} heard near a Twoleg place",
```
other thoughts use "Twoleg place", not "Twolegplace"; i'm not certain which one is corrected/intended, but suggesting editing it here for consistency